manifests/server.pp

```diff
@@ -58,6 +58,7 @@
     owner          => 'root',
     group          => 'root',
     mode           => '0400',
+    order          => 'numeric',
```

Comment:
According to the module, this is the default. Which version of `concat` is being used?

Reply:
@ayohrling pointed out this little gem that reverses the ordering when called via defined type https://github.com/puppetlabs/puppetlabs-concat/blob/master/manifests/init.pp#L63 :man_facepalming: